build.gradle

```diff
@@ -9,7 +9,6 @@ repositories {
 dependencies {
     implementation 'junit:junit:4.13.1'
```

Comment:
Make sure this build.gradle is up to date with TIm's pull request gradle files and that it still works. There was a junit version change I believe

Reply:
Yeah this should be changed to 4.13.2